AirCasting/CreateSessionViews/WifiPopupView.swift

```diff
@@ -66,6 +70,12 @@ struct WifiPopupView: View {
             isSSIDTextfieldDisplayed = true
         }
     }
+    
+    var confirmationButton: some View {
```

Comment:
What do you think about renaming it to `connectButton`? 
I'm not good with the names, it's just an idea, please ignore it if you prefer it this way :D